package.json

```diff
@@ -21,7 +21,8 @@
     "browser:dev": "tsx browser/src/index.ts",
     "prebrowser:build": "npm install",
     "browser:build": "tsc --project browser",
-    "browser:start": "node browser/dist/index.js"
+    "browser:start": "node browser/dist/index.js",
+    "healthcheck": "screen -ls 2>/dev/null | grep -E '(agent-|\\tagent\\t)' > /dev/null && echo 'Agent running' || ([ -f error.log ] && cat error.log || ([ -f browser-error.log ] && cat browser-error.log) || echo 'No Sockets found in /run/screen/S-root.')"
```

Comment:
This should run a script stored in scripts/healthcheck.ts. If there are no screens, then see if there's an error.log file and return that if so